### PR TITLE
Adjust date on mobile version

### PR DESCRIPTION
**Pull request by Brandonz07**

### CODE REVIEW

File: src/pages/queues/outlets/queuesInProgress/config/table.tsx
Comment (novadeveloper14) [inline src/pages/queues/outlets/queuesInProgress/config/table.tsx:1]: In the following location of the application *"src\components\data\Table\stories\mocks.tsx"* we have a file where the storybook of the table component is configured

(Storybook is a development tool used to build and document user interface (UI) components. It is not a complete framework, but rather a library that can be integrated into your existing project, whether you are using React, Vue, Angular, Svelte, or other JavaScript frameworks.). To deploy it in our project we must enter the following code in the console:

```
npm run storybook
```

Here if we enter the table component we realize that it is also showing the calendar so we should eliminate it in the same way as was done in the application in the array called actionsResponsiveMock eliminate the id date as follows:

image

```
const actionsResponsiveMock = [
  {
    id: "Status",
    actionName: "",
    content: () => <Icon appearance="danger" icon={<MdCancel />} size="20px" />,
  },
  {
    id: "Details",
    actionName: "Detalles",
    content: () => (
      <Icon appearance="dark" icon={<MdOutlineRemoveRedEye />} size="16px" />
    ),
  },
];
```
Don't forget to remove the import of the icon at the beginning of the file.

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/pages/queues/outlets/queuesInProgress/config/table.tsx
[the remark this answers is not in the document]
Comment (Brandonz07) [reply src/pages/queues/outlets/queuesInProgress/config/table.tsx:1]: remove the iMovil, and the icon from both `actionsResponsiveConfig` and `infoDataTable`

Comment (Brandonz07) [reply src/pages/queues/outlets/queuesInProgress/config/table.tsx:1]: remove icon in stories

**Review by novadeveloper14 — APPROVED** [review]

**Review by carin91 — CHANGES_REQUESTED** [review]
Your version lacks the borders. Adjust the app to the figma wireframes.
image

**Pull request merged**